game/app/scenes/data_manager.gd

```diff
@@ -86,7 +86,7 @@ class ScoreSorter:
 
 
 ## Handle data for the game that ended.
-func game_ended(player: String, game_id: String, start_time, score: Dictionary):
+func game_ended(player: String, game_id: String, start_time, score_or_null):
```

Comment:
Don't like the name. Is it score or is it null or is the score null?
Just call it score and add a comment with which values are expected.